native/src/components/OpeningEntry.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import { TimeSlot } from 'api-client/src/types'
+
+import { contentDirection } from '../constants/contentDirection'
+
+const EntryContainer = styled.View<{ language: string }>`
+  display: flex;
+  flex-direction: ${props => contentDirection(props.language)};
+  justify-content: space-between;
+  padding: ${props => (contentDirection(props.language) === 'row' ? '4px 36px 4px 0' : '4px 26px 4px 0')};
+`
+const Timeslot = styled.View`
+  display: flex;
+  flex-direction: column;
+`
+
+const TimeSlotEntry = styled.Text<{ isCurrentDay: boolean; notFirstChild?: boolean }>`
```

Comment:
not sure if its better readable because i set explicity style for `notFirstChild` so i would have to negate in the styled component
` ${props => !props.firstChild && 'margin-top: 8px'};`
Don't think its nicer tbh.